http/rest-client-reactive-vanilla/src/main/java/io/quarkus/ts/http/restclient/reactive/VersionHeaderFilter.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.ts.http.restclient.reactive;
```

Comment:
I think it should also contain _vanilla_ in package name because it is easier to identify Maven module just from package name when reading Jenkins.

I don't insist on this.

Reply:
You are right, it would be better for investigation. I've changed it.